src/challenge/Students.java

```diff
@@ -0,0 +1,24 @@
+package challenge;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Students {
+    public static void main(String[] args) {
+        ArrayList student = new ArrayList();
```

Comment:
When we create a list, the variable should be plural or with  `List` at the end. We can have in our case `students` or `studentList`.
You have declared in your code a list with `List<String> names = new ArrayList<>();`. This is the construction that we want to have. We can have type `List<dataTypeGoesHere>`. Declaring ArrayList student = ... is not something that we should do.

Valid constructions:
`List<String> stringArrayList = new ArrayList<>();`
`List<Integer> integerList = new ArrayList<>();`
`List<String> stringLinkedList = new LinkedList<>();`